client/hooks/trips/useAddTrip.ts

```diff
@@ -1,15 +1,19 @@
+import { useRouter } from 'expo-router';
 import { queryTrpc } from '../../trpc';
 
 export const useAddTrip = () => {
+  const router = useRouter();
   const utils = queryTrpc.useUtils();
   const mutation = queryTrpc.addTrip.useMutation();
 
   const addTrip = (newTrip) => {
     mutation.mutate(newTrip, {
       onSuccess: () => {
         utils.getTrips.invalidate();
+        router.push('/trips');
```

Comment:
same comment about id as above, although i wonder if this hook is the best place to be routing in 🤔 fine for now, might be only option without fully refactoring the hook